zebrabox_handlers/srx_flyscans.py

```diff
@@ -0,0 +1,13 @@
+from . import HandlerBase
+
+import h5py
+
+
+class ZebraHDF5Handler(HandlerBase):
+    HANDLER_NAME = 'ZEBRA_HDF51'
```

Comment:
This doesn't actually _do_ anything, but it's a convention to make it easy to see which spec a handler works with.

As @mrakitin recently pointed out, rightly, it's not great that we store this information separately in the `entry_points` specification and here, as they could get out of sync. But for the moment this is our pattern, so let's stick with it and circle back to consider the problem in general after the deployment.
```suggestion
    specs = {'ZEBRA_HDF51'}
```